client/src/services/share-point.ts

```diff
@@ -41,22 +32,24 @@ export type SiteActivityWithSites = Partial<SiteActivity & Site>
 
 export class SharePointService extends BaseService {
     public static async getAll(): Promise<Array<Site>> {
-        const { value } = await this.httpGet(graphLinks.sites)
-
+        const { value } = await graphClient.api('sites/getAllSites').get()
         return value
     }
 
     public static async getActivity(period: 30 | 90): Promise<Array<SiteActivity>> {
-        const { value } = await this.httpGet(graphLinks.sitesActivity(period))
-
+        const { value } = await graphClient.api(`/reports/microsoft.graph.getSharePointSiteUsageDetail(period='D${period}')?$format=application/json`).get()
```

Comment:
?$format=application/json is not needed

Reply:
If i remove the format i faced this error.

https://prnt.sc/rcAlAyuHKG1L